serializer.py

```diff
@@ -39,3 +28,97 @@ def usuario_from_db(usuario):
 
 def nome_usuario_from_web(**kwargs):
     return kwargs["nome_completo"] if "nome_completo" in kwargs else ""
+
+
+def genero_from_web(**kwargs):
+    return {
+        "nome": kwargs["nome"] if "nome" in kwargs else "",
+    }
+
+
+def genero_from_db(genero):
+    return {
+        "id": genero["id"],
+        "nome": genero["nome"]
+    }
+
+
+def nome_genero_from_web(**kwargs):
+    return kwargs["nome"] if "nome" in kwargs else ""
+
+
+def locacoes_from_web(**kwargs):
+    return {
+        "data_inicio": kwargs["data_inicio"] if "data_inicio" in kwargs else "",
+        "data_fim": kwargs["data_fim"] if "data_fim" in kwargs else "",
+        "filmes_id": kwargs["filmes_id"] if "filmes_id" in kwargs else "",
+        "usuarios_id": kwargs["usuarios_id"] if "usuarios_id" in kwargs else "",
+    }
+
+
+def locacoes_from_db(locacao):
+    return {
+        "id": locacao["id"],
+        "data_inicio": locacao["data_inicio"],
+        "data_fim": locacao["data_fim"],
+        "filmes_id": locacao["filmes_id"],
+        "usuarios_id": locacao["usuarios_id"]
+    }
+
+
+def data_locacoes_from_web(**kwargs):
+    return kwargs["data_inicio"] if "data_inicio" in kwargs else ""
+
+
+def filmes_from_web(**kwargs):
+    return {
+        "titulo": kwargs["titulo"] if "titulo" in kwargs else "",
+        "ano": kwargs["ano"] if "ano" in kwargs else "",
+        "classificacao": kwargs["classificacao"] if "classificacao" in kwargs else "",
+        "preco": str(kwargs["preco"]) if "preco" in kwargs else "",
+        "diretores_id": kwargs["diretores_id"] if "diretores_id" in kwargs else "",
+        "generos_id": kwargs["generos_id"] if "generos_id" in kwargs else "",
+    }
+
+
+def filmes_from_db(filmes):
+    return {
+        "id": filmes["id"],
+        "titulo": filmes["titulo"],
+        "ano": filmes["ano"],
+        "classificacao": filmes["classificacao"],
+        "preco": filmes["preco"],
+        "diretores_id": filmes["diretores_id"],
+        "generos_id": filmes["generos_id"]
+    }
```

Comment:
Acho que você recebe 
```suggestion
def filmes_from_db(filme):
    return {
        "id": filme["id"],
        "titulo": filme["titulo"],
        "ano": filme["ano"],
        "classificacao": filme["classificacao"],
        "preco": filme["preco"],
        "diretores_id": filme["diretores_id"],
        "generos_id": filme["generos_id"]
    }
```